packages/eslint-plugin-blade/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import noCrossPlatformImportsRule from './rules/no-cross-platform-imports';
```

Comment:
eslint-plugin-blade sounds like a eslint plugin that our consumers should use to lint blade errors. Can we add this plugin code without creating a separate package 🤔 ? Since its specific to our repo? or maybe a new open-source plugin `eslint-plugin-no-cross-platform-imports`?

Reply:
@saurabhdaware We are not publishing `eslint-plugin-blade` anywhere, this is for internal use only and not exposed to consumers.

Reply:
oh cool then

Reply:
Similar thoughts, what happens if later down the line we want to expose a consumer end eslint plugin for blade? 

Should we call our internal plugin `esling-plugin-blade-internal`?

Reply:
I understand the concern, but the likelihood of creating a consumer-facing eslint plugin is very low. We have proper types and TS checks in place for blade. We can rename the plugin later if required, also we can add as many rules as we want so maybe be can use the same plugin in the future.